### ISSUE TITLE
Do not split PDFs into pages before processing them in Document Intelligence

### ISSUE THREAD

**Issue by DFMERA** [Azure-Samples/azure-search-openai-demo-csharp#385]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [X] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Run the preperdocs process or index a new PDF

### Expected/desired behavior
> Do not split PDFs into pages before processing them in Document Intelligence

### OS and Version?
> Windows 11

### Mention any other details that might be useful
I noticed a difference in the document preparation process between this project and the Python project. The Python project does not split PDFs into pages before processing them in Document Intelligence, this makes sense because splitting into pages can lose context of paragraphs or tables across multiple pages. If you are okay with making this change in the project **I could make the code and the pull request.**

**Comment by dupeone** [comment]
I agree.  When I run the demo, and ask about values, It only lists the 6 on the page, but misses the 2 on the next page.
It loses context when split it seems.

**Comment by nm-free** [comment]
@DFMERA 
I agree.
